src/main/webapp/level2.html

```diff
@@ -9,6 +9,7 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
 </head>
 <body>
-    <p>"What animal do you not want to play cards with? (TIP: Upload a picture of the animal for your answer. Submit the file and then submit in the answer box.)"</p>
+    <p>What animal do you not want to play cards with?</p>
+    <p class="card">Upload a picture of the animal for your answer. Submit the file and then submit in the answer box.</p>
```

Comment:
this might work better if you say something like "Upload a picture of the animal corresponding to the answer to the following riddle. Submit the file, press save, then press submit when you think you have cracked it!"

Reply:
I made a shorter version, since I changed the first button to "upload" I think it will be clearer.